configs/default/policy.yaml

```diff
@@ -0,0 +1,27 @@
+#
+# To use this configuration:
+#   ec validate image ... --policy github.com/enterprise-contract//configs/default.yaml?ref=main
+#
+description: >
+  Use the policy rules from the "minimal" collection. This and other collections are defined in
+  https://redhat-appstudio.github.io/docs.stonesoup.io/ec-policies/release_policy.html#_available_rule_collections
+  The minimal collection is a small set of policy rules that should be easy to pass for brand new RHTAP users.
+  If a different policy configuration is desired, this resource can serve as a starting point. See the docs on
+  how to include and exclude rules at https://redhat-appstudio.github.io/docs.stonesoup.io/ec-policies/policy_configuration.html#_including_and_excluding_rules
+
+publicKey: "k8s://tekton-chains/public-key"
+
+sources:
+  - name: Default RHTAP Release Policies
```

Comment:
Should this say Enterprise Contract instead of RHTAP?

Reply:
I made it just "Default".